.changeset/dry-rules-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
+---
+
+Fixes cacheOnDemandPages setting to only cache GET/HEAD requests by default
```

Comment:
Since I'm here, I'll throw my hat into the ring! I agree with "updates", as nothing broke before!

```suggestion
Updates the default behavior of `cacheOnDemandPages` to cache only GET/HEAD requests 
```
Or you can place "default" back at the end. Up to you!